### PR TITLE
runtime: Fix more virtiofs args

### PR DESCRIPTION
**Pull request by gkurz**
Update some more virtiofsd arguments from the legacy syntax of the C implementation to the new syntax used by the rust implementation.

### CODE REVIEW

**Review by jepio — APPROVED** [review]

File: src/runtime/virtcontainers/virtiofsd_test.go [hunk 1]
```diff
@@ -130,12 +130,7 @@ func TestValid(t *testing.T) {
 		{"source is not available", func(v *virtiofsd) {
 			v.sourcePath = "/foo/bar"
 		}, errVirtiofsdSourceNotAvailable, nil},
-		{"replace cache mode none by never", func(v *virtiofsd) {
-			v.cache = "none"
-		}, nil, func(name string, a *assert.Assertions, v *virtiofsd) {
-			a.Equal("never", v.cache, "test case %+s, cache mode none should be replaced by never", name)
-		}},
-		{"invald cache mode: replace none by never", func(v *virtiofsd) {
+		{"invald cache mode", func(v *virtiofsd) {
```
Comment (stevenhorsman) [inline src/runtime/virtcontainers/virtiofsd_test.go:133]: `invald` -> `invalid` (which I know was there before, but worth fixing in this.

**Review by stevenhorsman — APPROVED** [review]
LGTM. Thanks

File: src/runtime/virtcontainers/virtiofsd_test.go (hunk 1 above)
Comment (gkurz) [reply src/runtime/virtcontainers/virtiofsd_test.go]: Good catch. Fixed. Thanks @stevenhorsman !

**Review by amshinde — APPROVED** [review]

**Pull request merged**